docs/rules/brace-style.md

```diff
@@ -40,27 +40,27 @@ While no style is considered better than the other, most developers agree that h
 
 ## Rule Details
 
-This rule is aimed at enforcing a particular brace style in JavaScript. As such, it warns whenever it sees a statement or declaration that does not adhere to the one true brace style.
+This rule is aimed at enforcing a particular brace style in JavaScript.
```

Comment:
Top be consistent with a previous rule in this PR, what if we structured the beginning of this sentence as such "This rule aims to [...]" such that this would become "This rule aims to enforce a particular brace style in JavaScript." and apply it the same way to other rules?